rhizome/host/e2e/spdk_rpc_e2e_spec.rb

```diff
@@ -0,0 +1,214 @@
+# frozen_string_literal: true
+
+require_relative "../lib/spdk_rpc"
+require_relative "../../common/lib/util"
+require "fileutils"
+
+RSpec.describe SpdkRpc do
```

Comment:
We could also all of these as a Prog, but since we are testing a library, we needed a command binary in `rhizome/host/bin` to send commands to the library. Then I thought why not have an rspec and run it directly instead of creating a binary?

This seemed simpler & have a faster development loop (can be run locally and as part of E2E CI)

Reply:
Okay, now your comments on the call made sense. No problem doing it this way, the relationship of rspec to spdk here is bit like the control plane to postgres. But as you were mentioning, it's a bit brutal to expect nearly-everyone  to have spdk set up.